server/src/test/java/io/crate/planner/operators/OuterJoinRewriteTest.java

```diff
@@ -121,4 +123,31 @@ public void testFilterOnFullOuterJoinIsPartiallyPushedDownToTheRightSide() {
             """;
         assertThat(plan).isEqualTo(expectedPlan);
     }
+
+    @Test
+    public void test_filter_on_aliased_nested_join_is_partially_pushed_down() {
+        var plan = sqlExecutor.logicalPlan(
+            """
+                select * from (
+                    select t1.x as t1x, t2.x as t2x
+                    from t1
+                    left join t2 on t1.x = t2.x
+                    left join t3 on t2.x = t3.x
+                ) as v
+                where t1x > 1 and t2x > 5
+                """
```

Comment:
Changing `t1x > 1 and t2x > 5` to `t1x > 1 OR t2x > 5` prevents the queries to be pushed down.

Reply:
Resolved by https://github.com/crate/crate/pull/15085/commits/70a9c657a825af401561344002cffd1689f3a2b7.